service-spec/service-spec-commons/src/main/java/io/hyscale/servicespec/commons/builder/Type.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.servicespec.commons.builder;
+
+/**
+ * Input type for service profile and spec
+ */
+public enum Type {
```

Comment:
Just type does not indicate what it is, clear on what entity that type is describing